web/templates/email/daily_digest.text.eex

```diff
@@ -1,31 +1,33 @@
-<%= gettext("# Daily digest of new Interests and Announcements") %>
+<%= gettext("# Daily digest of new Interests and Announcements\n") %>
 
 <%= unless Enum.empty?(@interests) do %>
 <%= gettext("Recently added interests") %>
--------------
+-----------------
 
 <%= for interest <- @interests do %>
   #<%= interest.name %> - <%= interest_url(Constable.Endpoint, :show, interest) %>
 <% end %>
 
 <% end %>
 <%= unless Enum.empty?(@announcements) do %>
-<%= gettext("Recently added announcements") %>
+<%= gettext("Recently added announcements\n") %>
 -----------------
 
 <%= for announcement <- @announcements do %>
   <%= announcement.title %> - <%= announcement_url(Constable.Endpoint, :show, announcement) %>
   <%= gettext("posted by") %> <%= announcement.user.name %> <%= gettext("in") %> <%= interest_names(announcement) %>
+  <%= gettext("\n") %>
```

Comment:
agreed, thanks @SViccari and @paulcsmith ... I think we can get away with just the newline character `\n` without the need for the carriage return character `\r`.

Fixed in 0ebf7f3